File: C++/college/student.cpp
#include<iostream>
#include<string>
using namespace std;
class Student {
    string name;
    string classname;
    int roll;
    public:
    void setName(string n, string c, int r) ;
    void display();
};
void Student::setName(string n, string c, int r){
    name = n;
    classname = c;
    roll = r;
}
void Student::display(){
    cout<<name<<endl<<classname<<endl<<roll<<endl;
}
int main(){
    Student s;
    s.setName("roopali sharma","CSE",61);
    s.display();
    Student s1;
    s1.setName("moon","cse",62);
    s1.display();
    

}

File: JAVA/oops/student.java
class students{
    String name;
    String classname;
    int roll;
    students(String n,String c,int r){
        name = n;
        classname = c;
        roll = r;
    }
    void display(){
        System.out.println(name);
        System.out.println(classname);
        System.out.println(roll);
    }
    }
public class student {
    public static void main(String[] args) {
        students s1= new students("roopali","cse",61);
        s1.display();
        System.out.println("---------");
        students s2 = new students("moon","cse",62);
        s2.display();
    }
    
}
